Apps/W1/HybridGP/app/src/pages/GPMigrationGeneralSettings.Page.al

```diff
@@ -0,0 +1,57 @@
+page 4102 "GP Migration General Settings"
```

Comment:
If you agree, I think we should leave the 2 step out of this PR and keep the code running as it is. Lets align the code once the troubleshooting changes are in. I believe we will need to churn more code, so it would be good to align the changes.

Reply:
Will do